src/openrct2/interface/InteractiveConsole.cpp

```diff
@@ -214,7 +214,7 @@ static int32_t cc_rides(InteractiveConsole& console, const arguments_t& argv)
                 else
                 {
                     auto ride = get_ride(ride_index);
-                    if (mode <= 0 || mode > (RIDE_MODE_COUNT - 1))
+                    if (mode <= 0 || mode > (static_cast<uint8_t>(RideMode::Count) - 1))
```

Comment:
```suggestion
                    if (mode <= 0 || mode >= static_cast<uint8_t>(RideMode::Count))
```

Reply:
Also I'm not sure why 0 would be an invalid mode, 0 is RideMode::Normal, so this might be a bug.

Reply:
I'm quite sure it is, so the check would have to be for `< 0` (assuming RideMode is signed, otherwise it should be taken out entirely).

Reply:
I've changed this check to
```
if (mode >= static_cast<uint8_t>(RideMode::Count))
                    {
                        console.WriteFormatLine("Invalid ride mode.");
                    }
                    else
```